linchpin/provision/rundb.yml

```diff
@@ -19,7 +19,7 @@
         conn_str: "{{ rundb_path }}"
         operation: init
         table: "{{ rundb_table }}"
-        value: {'inputs': [], 'outputs': [], 'start': '', 'end': '', 'rc': 0, 'action': ''}
+        value: {'inputs': [], 'outputs': [], 'topology_ouputs': [], 'start': '', 'end': '', 'rc': 0, 'action': '', 'inventory_path': ''}
```

Comment:
topology_outputs is spelled incorrectly here. Plus, shouldn't this be `resources`? If we're going to be backward compatible, we should at least use the same reference point in the code and data store.